cfy_manager/networks/networks.py

```diff
@@ -49,7 +62,8 @@ def _update_metadata_file(metadata, networks):
     :param networks: a dict containing the new networks
     """
 
-    old_networks = metadata['network_names']
+    old_networks = metadata['network_names'] if metadata.get(
```

Comment:
```suggestion
    old_networks = metadata.get('network_names') or []
```
In python, `dict.get(key)` returns `None` if the key is not present in the dictionary. The syntax `x = y or z` means: 

```
if y is None:
  x = z
else:
  x = y
```

Reply:
Thats right but I I do not want to return a None since I need a list. Thats the reason why I have else []

Reply:
The way I suggested would do the same thing.

Reply:
What I need from this is even to lookup the value from metadata and if it does not contains anything then I need just to fallback to empty list, thats the whole point of this. I still did not get your point here

Reply:
As I wrote on slack, a better line would be `metadata.get('network_names', [])`